src/test/java/BaseTest.java

```diff
@@ -1,11 +1,69 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
+import java.time.Duration;
+
 public class BaseTest {
 
+    public static WebDriver driver = null;
+    public static WebDriverWait wait = null;
+
     @BeforeSuite
-    static void setupClass() {
-        WebDriverManager.chromedriver().setup();
+    static void setupClass() {WebDriverManager.chromedriver().setup();}
+
+    @BeforeMethod
+    public void launchBrowser() {
+        ChromeOptions options = new ChromeOptions();
+//        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--disable-notifications", "--remote-allow-origins=*", "--incognito", "--start-maximized");
+        options.setExperimentalOption("excludeSwitches", new String[]{"enable-automation"});
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        String url = "https://qa.koel.app/"; // Задание URL-адреса, который будет открыт в браузере
+        driver.get(url); // Открытие указанной URL-страницы
+
+    }
+
+    @AfterMethod
+    public void closeBrowser() {driver.quit(); }
+
+
+    protected void clickSubmit() throws InterruptedException {
+        WebElement buttonLogIn = driver.findElement(By.cssSelector("[type='submit']"));
+        buttonLogIn.click();
+        Thread.sleep(2000);
+
+    }
+
+    protected void enterPassword(String password) {
+        WebElement logPassword = driver.findElement(By.cssSelector("[type='password']"));
+        logPassword.click();
+        logPassword.clear();
+        logPassword.sendKeys(password);
+
+    }
+
+    protected void enterEmail(String email) {
+        WebElement logEmail = driver.findElement(By.xpath("//input[@type='email']"));
+        logEmail.click();
+        logEmail.clear();
+        logEmail.sendKeys(email);
```

Comment:
Page Object Model (POM): recommended to associate the methods with the corresponding page class where they are relevant. In the POM, each page or section of a web application is represented by a separate class. These classes encapsulate the elements and actions specific to that page or section.

Move these methods to the corresponding class according to the POM.

For example
![Screenshot_2](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/b07cd7c1-7977-4e53-80ea-8045ac17f779)